library.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "unzipLIB",
+    "version": "1.0.0",
+    "license": "Apache-2.0",
+    "homepage": "https://github.com/sindarin-inc/unzipLIB/",
+    "frameworks": "*",
+    "platforms": "*",
+    "build": {
+        "libArchive": true
+    },
+    "dependencies": {
+        "jkirsons/zlib-PIO": "https://github.com/jkirsons/zlib-PIO.git#b9aaf78"
```

Comment:
This was an attempt to fix the platformio sim build automatically but it didn't work. Not sure why